### ISSUE TITLE
__init__() got an unexpected keyword argument 'allow_abbrev'

### ISSUE THREAD

**Issue by jjnilton** [bharadwaj-raju/TextSuggest#47]
I'm trying to use TextSuggest in a different system now, the OS is Lubuntu 14.04, but when I try to use the `textsuggest` command I always get the error below:

```
$ rofi -v
Version: 0.15.11
$ xdotool --version
xdotool version 3.20140217.1
$ xsel --version
xsel version 1.2.0 by Conrad Parker <conrad@vergenet.net>
```

```
$ textsuggest --version (cloned today from repo)
Traceback (most recent call last):
  File "/usr/bin/textsuggest", line 59, in <module>
    - Full README: /usr/share/doc/textsuggest/README'''.replace('\t', '').replace('    ', ''))
TypeError: __init__() got an unexpected keyword argument 'allow_abbrev'
```

**Comment by bharadwaj-raju** [comment]
I suppose 14.04 has a too old Python version. `allow_abbrev` is a relatively recent feature.

Please upgrade Python.

**Comment by jjnilton** [comment]
@bharadwaj-raju that's right. Seems `allow_abbrev` is available from Python 3.5, and the version available in the Ubuntu repository is 3.4.2. Maybe it could be listed as a requirement.

```
$ python3 --version
Python 3.4.3
```

Thanks!

**Comment by bharadwaj-raju** [comment]
I'll update the README or better: just add a `try`-`except` clause to it.

**Comment by bharadwaj-raju** [comment]
Fixed as of commit 5548e2870198.
